src/main/java/seedu/address/logic/commands/StarCommand.java

```diff
@@ -0,0 +1,91 @@
+package seedu.address.logic.commands;
+
+import static java.util.Objects.requireNonNull;
+import static seedu.address.logic.parser.CliSyntax.PREFIX_STAR;
+
+import java.util.List;
+
+import seedu.address.commons.core.index.Index;
+import seedu.address.commons.util.ToStringBuilder;
+import seedu.address.logic.Messages;
+import seedu.address.logic.commands.exceptions.CommandException;
+import seedu.address.model.Model;
+import seedu.address.model.person.Person;
+import seedu.address.model.person.Star;
+
+/**
+ * Adds a star to a student in the address book.
+ * @code star 1 s/3
+ */
+public class StarCommand extends Command {
+
+    public static final String COMMAND_WORD = "star";
+
+    public static final String MESSAGE_USAGE = COMMAND_WORD + ": Edits the remark of the person identified "
```

Comment:
I think you forgot to change the `MESSAGE_USAGE` for the star command, seems like you copied from the remark command tutorial.

Reply:
Will update.